contracts/pods/PodFactory.sol

```diff
@@ -270,34 +258,36 @@ contract PodFactory is CoreRef, IPodFactory {
         return podController.podIdToSafe(podId);
     }
 
-    /// @notice Create an optimistic timelock, linking to an Orca pod
+    /// @notice Create a timelock, linking to an Orca pod
     /// @param safeAddress Address of the Gnosis Safe
     /// @param minDelay Delay on the timelock
     /// @param publicExecutor Non-permissioned smart contract that
     ///        allows any address to execute a ready transaction
-    /// @param vetoController Address which manages veto rights over a pod timelock
+    /// @param podAdmin Address which is the admin of the Orca pods
     ///        Will also be able to propose
-    function createOptimisticTimelock(
+    function _createTimelock(
         address safeAddress,
         uint256 minDelay,
         address publicExecutor,
-        address vetoController
+        address podAdmin
     ) internal returns (address) {
-        address[] memory proposers = new address[](2);
+        address[] memory proposers = new address[](2); // cancel timelock
         proposers[0] = safeAddress;
-        proposers[1] = vetoController;
+        proposers[1] = podAdmin;
 
         address[] memory executors = new address[](2);
         executors[0] = safeAddress;
         executors[1] = publicExecutor;
 
-        OptimisticTimelock timelock = new OptimisticTimelock(
-            address(core()),
+        TimelockController timelock = new TimelockController(
             minDelay,
             proposers,
             executors
         );
-        emit CreateOptimisticTimelock(address(timelock));
+        // Revoke TIMELOCK_ADMIN_ROLE priviledges from deployer factory
+        timelock.revokeRole(timelock.TIMELOCK_ADMIN_ROLE(), address(this));
```

Comment:
Safety/standard practice, even though this factory has no method to take advantage of this role